projects/admin/src/app/routes/route-tool.service.ts

```diff
@@ -211,6 +211,34 @@ export class RouteToolService {
     });
   }
 
+  /**
+   * Check all permissions of the record
+   *
+   * @param record - Object: the resource object to check
+   * @param recordType - String: the record type
+   * @return Observable providing object with 2 attributes :
+   *     - 'can' - Boolean: to know if the resource could be deleted
+   *     - 'message' - String: the message to display if the record cannot be deleted
```

Comment:
The explication can be more generalistic, not specific to deletion.

Reply:
done